projects/plugins/jetpack/changelog/earn-fix-payments-intro-blocks-true

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: bugfix
+Comment: one line change -- should be obvious
```

Comment:
Approving, but please do put something here that's useful. The changelogs are used outside of the context of the PR itself so, in and of itself, this changelog is not obvious.

Reply:
Oops, sorry!